### PR TITLE
Updates based on Marc and Burton's comments

### PR DESCRIPTION
**Pull request by haoming-quantil**
Updates based on Marc and Burton's comments

### CODE REVIEW

File: docs/portal/reports.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Reports
 
-HDT Reports give you information about traffic to your HDT transports, covering "Traffic Volume", "Traffic Bandwidth" and "Requests"
+HDT Reports provides information about traffic to your HDT transports. Reports cover traffic volume, traffic bandwidth, and requests.
```
Comment (marcfishelquantil) [inline docs/portal/reports.md:3]: Please replace "provides" with "provide."

File: docs/portal/transports/add-transport.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Add New Transport
 
-To create a transport for your High-speed Data Transmission service, click the **Add New Transport** button at the top right of the <strong> Transports </strong> page,  then the wizard instructions appears.
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of the HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:3]: Please insert the word "the" between "of" and "HDT."

File: docs/portal/transports/add-transport.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+To add a new transport, click the **Add New Transport** button at the top-right of the <strong> Transports </strong> page to display the wizard instructions.
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:4]: Please remove the hyphen between "top-right."

File: docs/portal/transports/add-transport.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 ## Basic Info
 ![null](</docs/resources/images/transports/add-transport-basic-info.png>)
 
 | Fields               | Description   |
 | -------------------- | ------------- |
-| Transport Name       | Enter a name to identify this transport. If you omit this parameter, the HDT portal generates a default name in the format of "Target Domain:Target Port|
+| Transport Name       | Enter a name to identify this transport. If you omit this parameter, the HDT portal will generate a default name in the format of "Target Domain:Target Port|
 | Version Number       | Version number of the transport configuration in history. |
-| Application Protocol | <li><strong> HTTP </strong> : For HTTP protocol.</li> <li><strong> HTTPS </strong> : For HTTPS protocol with SNI(Server Name Indication). |
+| Application Protocol | <li><strong> HTTP </strong>: For HTTP protocol.</li> <li><strong> HTTPS </strong>: For HTTPS protocol with SNI(Server Name Indication). |
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:13]: Please insert a space between "SNI" and "(Server."

File: docs/portal/transports/add-transport.md [hunk 5]
```diff
@@ -20,15 +20,15 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
 | Fields               | Description   |
 | -------------------- | ------------- |
 | Fixed Shields        | The shield servers (the HDT servers that connection to your origin servers) will be fixed. |
-| Dedicated IP Service | The IPs for serving the transport will be dedicated, and only for the cname. The option will be NOT applicable if you select HTTP or HTTPs as the Application Protocol. If you wish to use <strong> Dedicated IP Service </strong> for your HTTP/HTTPs applications, please select Others as the Application Protocol.|
+| Dedicated IP Service | The IP addresses for serving the transport will be dedicated, and only for the cname. The option will NOT be applicable if you select HTTP or HTTPs as the Application Protocol. To use <strong> Dedicated IP Service </strong> for your HTTP/HTTPs applications, select Others as the Application Protocol.|
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:23]: Please consider changing the first sentence to: "The IP addresses for serving the transport is dedicated to the cname only."
Should "cname" be capitalized here and wherever else it is used?
In the second sentence, please change the capitalization from "HTTPs" to "HTTPS."

File: docs/portal/transports/add-transport.md [hunk 6]
```diff
@@ -20,15 +20,15 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
...
...r into the request to carry the client’s real IP address for HTTP and HTTPs application. (For HTTPs applications, the client certificate must be deployed at the HDT servers. Please contact HDT support team if you need to deploy a client certificate.) </li> <li><strong> TCP Option 0x4e </strong>: HDT establishes a TCP connection with the origin server and sends the client's real IP address to the origin server through the tcp-option 0x4e in the TCP message; the origin server can obtain the client's real IP address through the F5 device or the Linux kernel module provided by HDT.</li> <li><strong>proxy protocol v1</strong>: Using proxy protocol v1, HDT establishes a TCP connection with the origin server, and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v1 message.</li><li><strong>proxy protocol v2 </strong>: Using proxy protocol v2, HDT establishes a TCP connection with the origin server, and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v2 message.</li>|
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:30]: Please  change the capitalization from "HTTPs" to "HTTPS" in two places. In the first sentence, please change "application" to "applications."
Please add the word the to the second sentence as follows: "Please contact the HDT support team..."

File: docs/portal/transports/add-transport.md [hunk 7]
```diff
@@ -20,15 +20,15 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
...
+|Transfer Strategy     |<li><strong> Default </strong>: Low cost and high efficiency. </li> <li><strong> Interactive </strong>: For applications requiring better response time, HDT internal routing is enabled in this mode. </li> <li><strong> High Concurrency </strong>：Suitable for small data transfers and high concurrency. Normally, concurrency is considered to be high when its value exceeds 1,000 connections and low when its value is below than 100 connections. </li> <li><strong> High Real-time </strong>: Suitable for applications that deliver small data with low concurrency (fewer than 100 connections) and require optimized response time. </li> |
```
Comment (marcfishelquantil) [inline docs/portal/transports/add-transport.md:31]: In the second sentence, please change "response time" to "response times."
In the sentence"...when its value is below than 100 connections."" please remove the word "than."
In the last sentence, please consider replacing "and require optimized response time." with "and require optimized response times."

File: docs/portal/transports/delete-transport.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Delete a Transport
 
-To delete a transport, search the transport you want to delete in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Delete".
```
Comment (marcfishelquantil) [inline docs/portal/transports/delete-transport.md:3]: In the text string "...and select the "Delete"" please remove the word "the."

File: docs/portal/transports/delete-transport.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
...
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/delete-transport.md:3]: Please add the word "the" as follows: "... from the left pane of the HDT portal."

File: docs/portal/transports/delete-transport.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
+To delete a transport, click the "Action" button of the transport entry in the list and select "Delete". If the transport was not in the current page, you can search it in the search box.
```
Comment (marcfishelquantil) [inline docs/portal/transports/delete-transport.md:4]: Please change "search it" with "search for it."
Please apply this suggestion in other areas where "search it" appears.

File: docs/portal/transports/edit-transport.md [hunk 11]
```diff
@@ -1,11 +1,11 @@
 # Edit a Transport
 
-To edit a transport, search the transport you want to edit in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Edit".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/edit-transport.md:3]: Please insert the word "the" between "of" and "HDT."

File: docs/portal/transports/edit-transport.md [hunk 12]
```diff
@@ -1,11 +1,11 @@
...
+To edit a transport, click the "Action" button of the transport entry in the list and select "Edit". If the transport was not in the current page, you can search it in the search box.
```
Comment (marcfishelquantil) [inline docs/portal/transports/edit-transport.md:4]: Another instance of "search it." I suggest replacing all instances of this string with "search for it."

File: docs/portal/transports/edit-transport.md [hunk 13]
```diff
@@ -1,11 +1,11 @@
...
 
 ![null](</docs/resources/images/transports/edit-transport-1.png>)
 
-Edit the items to make the changes you want to make, then click the "Next" button in the top right corner of the page.
+Edit the items to make the changes you want to make, then click the "Next" button in the top-right corner of the page.
```
Comment (marcfishelquantil) [inline docs/portal/transports/edit-transport.md:8]: Please consider changing "Edit the items to make the changes you want to make..." to "Edit the items as desired..."
If this string appears elsewhere, please apply this same edit for consistency.

File: docs/portal/transports/manage-transport.md [hunk 14]
```diff
@@ -7,9 +7,9 @@ To display this page, click **Transports** in the left pane.
 
 - To **filter transports**, type characters in the search box and then press the Enter key or click Magnifier icon. All transports that do not contain the typed characters will be hidden. Filtering is not case-sensitive. To remove the filter, click the "Reset" button.
 
-- To **view transport**, click the transport name allow you to view detailed configuration about the transport.
+- To **view transport**, click the transport name to display detailed configuration information about the transport.
 
-- To **Edit, Show History, Delete, Active/Suspend** a transport, click the "Action" dropdown box, and select the action you want to take.
+- To **Edit**, **Show History**, **Delete**, **Active**, or **Suspend** a transport, click the "Action" dropdown box, and select the action you want to take.
```
Comment (marcfishelquantil) [inline docs/portal/transports/manage-transport.md:12]: Please hyphenate "dropdown" as follows: "drop-down."
Ref: https://sites.udel.edu/it-style/vocabulary-guidelines/#:~:text=drop%2Ddown%20is%20hyphenated%20when%20used%20as%20an%20adjective.&text=drop%2Ddown%20menu%20(aka%20drop,when%20used%20as%20an%20adjective.

Please consider changing "you want to take" with "you want to perform."

File: docs/portal/transports/show-history.md [hunk 15]
```diff
@@ -1,10 +1,10 @@
 # Show History of a Transport
 
-To see history of a transport, search the transport you want to show in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Show History".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/show-history.md:3]: Please insert the word "the" between "of" and "HDT."

File: docs/portal/transports/show-history.md [hunk 16]
```diff
@@ -1,10 +1,10 @@
...
+To view history of a transport, click the "Action" button of the transport entry in the list and select "Show History". If the transport was not in the current page, you can search it in the search box.
```
Comment (marcfishelquantil) [inline docs/portal/transports/show-history.md:4]: Please insert the word "the" between "view" and "history."
Please change "search it" with "search for it."

File: docs/portal/transports/suspend-activate-transport.md [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # Suspend or Activate a transport
 
-To Suspend or Activate a transport, search the transport you want to make the change in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Suspend" or "Activate".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/suspend-activate-transport.md:3]: Please insert the word "the" between "of" and "HDT."

File: docs/portal/transports/suspend-activate-transport.md [hunk 18]
```diff
@@ -1,7 +1,7 @@
...
+To suspend or activate a transport, click the "Action" button  of the transport entry in the list and select "Suspend" or "Activate". If the transport was not in the current page, you can search it in the search box.
```
Comment (marcfishelquantil) [inline docs/portal/transports/suspend-activate-transport.md:4]: Please change "search it" with "search for it."

File: docs/portal/transports/view-transport.md [hunk 19]
```diff
@@ -1,6 +1,6 @@
 # View a Transport
 
-To view a transport, search the transport you want to view in the <strong> Transport </strong> page if it was not in the page, and then click the "Transport Name".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```
Comment (marcfishelquantil) [inline docs/portal/transports/view-transport.md:3]: Please insert the word "the" between "of" and "HDT."

File: docs/portal/transports/view-transport.md [hunk 20]
```diff
@@ -1,6 +1,6 @@
...
+To view a transport, click the "Transport Name" of the transport entry in the list. If the transport was not in the current page, you can search it in the search box.
```
Comment (marcfishelquantil) [inline docs/portal/transports/view-transport.md:4]: Please change "search it" with "search for it."

**Comment by marcfishelquantil** [comment]
Haoming, I reviewed and commented on areas of suggested improvement. Thank you for engaging me on this effort. Please let me know whether you have questions.

File: docs/portal/transports/suspend-activate-transport.md (hunk 18 above)
Comment (burtonquantil) [reply docs/portal/transports/suspend-activate-transport.md:4]: I'd also suggest  "If the transport **is** not in the current page"

File: docs/portal/transports/view-transport.md (hunk 20 above)
Comment (burtonquantil) [reply docs/portal/transports/view-transport.md:4]: I'd also suggest  replacing "was" with "is"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I added some suggestions.

File: apispec/openapi.json [hunk 21]
```diff
@@ -1817,7 +1817,7 @@
             }
           }
         },
-        "description": "Modify a transport according to {transport-id}, which is a transport id. If transport type of this transport is ipport, targetDomain, targetPort, transportName, useFtp, useUdp, useStandardPort, accessDomain, status, comments and ipWhiteList can be modified. If transport type of this transport is vpn or socks5, transportName, useFtp, user password, status, comments and ipWhiteList can be modified. It will return parameter information, which takes effect on the HDT system.",
+        "description": "Modify a transport according to {transport-id}, which is a transport ID. If transport type of is \"ipport\", then \"targetDomain\", \"targetPort\", \"transportName\", \"useFtp\", \"useUdp\", \"useStandardPort\", \"accessDomain\", \"status\", \"comments\", and \"ipWhiteList\" can be modified. If transport type of this transport is \"vpn\" or \"socks5\", \"transportName\", \"useFtp\", \"userPwd\", \"status\", \"comments\", and \"ipWhiteList\" can be modified. It will return parameter information, which takes effect on the HDT system.",
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:1820]: Yes, how about  "For successful requests, the settings applied to the HDT system will be returned in the response. "

File: apispec/openapi.json [hunk 22]
```diff
@@ -2357,7 +2357,7 @@
             }
           }
         },
-        "description": "Delete a transport according to {transport-id}, which is a transport id, and disassociate it with the calling customer. ",
+        "description": "Delete a transport according to {transport-id}, which is a transport ID, and disassociate it with the calling customer. ",
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:2360]: How about "Delete a transport according to the transport ID."

File: apispec/openapi.json [hunk 23]
```diff
@@ -4337,7 +4337,7 @@
...
+        "description": "Returns transport logs for a specified transport and time period. After getting the response, you can download the raw log from the returned URL.\nSample: \nwget  ‘http://logdl.mileweb.com/hdtlog/1/5067/20180610-1600-1659/35.186.212.62-443_2018-06-10-1600-1659_mproxy_0.log.gz?wskey=a26efbc6e29bdc30f091eb201dbc6236’ -O 1.gz\nCurrently, logs have a delay of 2 hours. This means that if you download a log two hours ago, the completeness would be about 99%. The log is aggregated hourly or daily, which is configurable for each transport ID (the configuration item is called log merge granularity). There may be more than one log for the same time slice, including late logs. For example, if a transport’s log merge granularity is hourly, there may be more than one log for each hour.\nWhether the log is sorted depends on the transport’s sorting configuration. Log sorting means that the log is sorted by the datetime in the log lines.\nThe maximum download speed for a single connection is limited to 5MB/s now.\n",
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:4340]: How about "Currently, there might be a 2 hours delay in data ingestion, for a log two hours eariler, the data accuracy is 99% approximately."

File: apispec/openapi.json [hunk 24]
```diff
@@ -5236,23 +5230,23 @@
       "transport.cName": {
         "type": "string",
         "title": "cName",
-        "description": "The cName HDT generates for a transport. Customer could use the cName to redirect the traffic to the HDT platform.",
+        "description": "The CNAME record HDT generates for a transport. Customer could use the CNAME record to redirect the traffic to the HDT platform.",
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:5233]: how about "The CNAME record HDT generates for a transport, which used to redirect the traffic to the HDT platform."

File: apispec/openapi.json (hunk 22 above)
[the remark this answers is not in the document]
Comment (burtonquantil) [reply apispec/openapi.json:2360]: @haoming-quantil  I think "by" or "using" is better than 'according to"

File: apispec/openapi.json (hunk 21 above)
[the remark this answers is not in the document]
Comment (burtonquantil) [reply apispec/openapi.json:1820]: @haoming-quantil  that's OK too.

File: apispec/openapi.json (hunk 23 above)
[the remark this answers is not in the document]
Comment (burtonquantil) [reply apispec/openapi.json:4340]: @haoming-quantil  Can we just simplify to: "There can be up to two hours delay in processing data. "   ?

Or do you think we really need to talk about the 99% accuracy?

[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:4340]: Yes, let's skip this 99% thing.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
There is just one grammar issue I noticed in the latest edits.

File: apispec/openapi.json [hunk 25]
```diff
@@ -5274,31 +5274,31 @@
...
         "title": "speedLimit",
-        "description": "Indicates the speed limit in Mbps. ",
+        "description": "Speed limit in Mbps. ",
         "type": "number"
       },
       "transport.concurrentLimit": {
         "title": "concurrentLimit",
         "type": "integer",
-        "description": "Indicates the concurrent limit."
+        "description": "Concurrent limit."
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply apispec/openapi.json:5291]: it is for the concurrent connections. so how about "The number of concurrent connections allowed for the transport."?

File: apispec/openapi.json [hunk 26]
```diff
@@ -5481,17 +5600,51 @@
...
         "title": "total",
         "type": "integer",
-        "description": "Indicates total request count at this period time"
+        "description": "Total count of requests at this period time"
       },
       "report.successRequestsCount": {
         "title": "success",
         "type": "integer",
-        "description": "Indicates total successful request count at this period time."
+        "description": "Total count of successful requests for a specified period of time."
```
Comment (burtonquantil) [inline apispec/openapi.json:5647]: "Total number of successful requests during the time period."

File: apispec/openapi.json [hunk 27]
```diff
@@ -5452,22 +5532,61 @@
...
         "title": "total-summary",
         "type": "integer",
-        "description": "indicates total request count for all records."
+        "description": "Total count of requests for all records."
       },
       "report.successRequestsSummary": {
         "title": "success-summary",
         "type": "integer",
-        "description": "indicates total successful request count for all records."
+        "description": "Total count of successful requests for all records."
```
Comment (burtonquantil) [inline apispec/openapi.json:5584]: "Total number of successful requests for all records."

File: apispec/openapi.json [hunk 28]
```diff
@@ -5452,22 +5532,61 @@
...
+              }
+            ]
+          }
         }
       },
       "report.totalRequestsSummary": {
         "title": "total-summary",
         "type": "integer",
-        "description": "indicates total request count for all records."
+        "description": "Total count of requests for all records."
```
Comment (burtonquantil) [inline apispec/openapi.json:5579]: Total number of requests for all records.

File: apispec/openapi.json [hunk 29]
```diff
@@ -5518,17 +5671,26 @@
...
+        "description": "URL of for downloading log file. The format of log file name are following:\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.log.gz\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.late.log.gz:\n1) domain-port represents the HDT transport;\n2) YYYY-MM-DD-hhmm-hhmm means the time range is from YYYY-MM-DD-hhmm to YYYY-MM-DD-hhmm;\n3) mproxy is a fixed string;\n4) late.log.gz refers to a log delay time is more than 2 hours. Usually, the proportion of the late log is small (no more than 1%).\n5) X is the sequence number. It increases when the uncompressed file size reaches 2GB.\n",
```
Comment (burtonquantil) [inline apispec/openapi.json:5690]: Change "URL of for downloading log file" to "URL for downloading log file"
and change "The format of log file name are following" to
"The log file name follows this format:"

File: docs/portal/transports/edit-transport.md [hunk 30]
```diff
@@ -1,11 +1,11 @@
...
 
-To edit a transport, search the transport you want to edit in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Edit".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of the HDT portal.
+To edit a transport, click the "Action" button of the transport entry in the list and select "Edit". If the transport is not in the current page, you can search for it in the search box.
 
 ![null](</docs/resources/images/transports/edit-transport-1.png>)
 
-Edit the items to make the changes you want to make, then click the "Next" button in the top right corner of the page.
+Edit the items as desired, then click the "Next" button in the top right corner of the page.
```
Comment (burtonquantil) [inline docs/portal/transports/edit-transport.md:8]: "as desired" is unnecessary.   The customer is always going to do what they want or need. I'd simplify to:  "Edit the settings and then click the "Next" button in the top right corner of the page."

File: docs/portal/transports/manage-transport.md [hunk 31]
```diff
@@ -7,9 +7,9 @@ To display this page, click **Transports** in the left pane.
 
 - To **filter transports**, type characters in the search box and then press the Enter key or click Magnifier icon. All transports that do not contain the typed characters will be hidden. Filtering is not case-sensitive. To remove the filter, click the "Reset" button.
 
-- To **view transport**, click the transport name allow you to view detailed configuration about the transport.
+- To **view transport**, click the transport name to display detailed configuration information about the transport.
 
-- To **Edit, Show History, Delete, Active/Suspend** a transport, click the "Action" dropdown box, and select the action you want to take.
+- To **Edit**, **Show History**, **Delete**, **Active**, or **Suspend** a transport, click the "Action" drop-down box, and select the action you want to perform.
```
Comment (burtonquantil) [inline docs/portal/transports/manage-transport.md:12]: Minor:  You can omit the words "you want"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
made a few small suggestions.

File: docs/intro.md [hunk 32]
```diff
@@ -1,17 +1,17 @@
 # Welcome to the High-Speed Data Transmission Customer Portal
 
-The High-Speed Data Transmission (HDT) Customer Portal is a web-based user interface for the HDT platform, where you can add, view, and manage your transports and retrieve detailed reports about the usage.
+The High-Speed Data Transmission (HDT) Customer Portal is a web-based user interface for the HDT platform where you can add, view, and manage your transports and retrieve detailed reports about the usage.
```
[the remark this answers is not in the document]
Comment (haoming-quantil) [reply docs/intro.md:3]: Yes, sounds right.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**